homeassistant/components/notify/__init__.py

```diff
@@ -32,6 +34,13 @@
 
 SERVICE_NOTIFY = "notify"
 
+NOTIFY_SERVICE_SCHEMA = vol.Schema({
+    vol.Required(ATTR_MESSAGE): cv.template,
+    vol.Optional(ATTR_TITLE, default=ATTR_TITLE_DEFAULT): cv.string,
+    vol.Optional(ATTR_TARGET): cv.string,
+    vol.Optional(ATTR_DATA): dict,      # nobody seems to be using this (yet)
```

Comment:
I wonder if we should remove the 'data' attribute. No existing notify platform uses it and so we don't even really know what to validate this as.